test/org/sagebionetworks/bridge/dynamodb/DynamoScheduledActivityDaoMockTest.java

```diff
@@ -189,12 +337,17 @@ public void testOnlyPersistedActivitiesReturned() {
             .withEvents(events).build();
 
         List<ScheduledActivity> activities = TestUtils.runSchedulerForActivities(context);
-        // Only mock the return of one of these activities
-        mockMapperResults(Lists.newArrayList(activities.get(0)));
```

Comment:
We should revert to using mockMapperResults(). These tests look fairly complicated, and I don't want to risk losing coverage for something that's supposed to be a simple test refactor.

Reply:
Okay, I reverted and started over, changing none of the existing mocking, and then adding in new tests and some code necessary to prevent UnnecessaryStubbingExceptions.